src/NUnitFramework/nunitlite.tests/CommandLineTests.cs

```diff
@@ -172,7 +172,7 @@ public void CanRecognizeLowerCaseOptionValues(string propertyName, string option
 
             foreach (string canonicalValue in canonicalValues)
             {
-                string lowercaseValue = canonicalValue.ToLower(CultureInfo.InvariantCulture);
+                string lowercaseValue = canonicalValue.ToLower();
```

Comment:
Shouldn't you use ToLowerInvariant(), which I believe is in .Net Standard (at least the documentation says it is.